### PR TITLE
removed unused code

### PR DESCRIPTION
**Pull request by nickywongdong**
also made some comments, let me know what you guys think

### CODE REVIEW

File: src/main/java/edu/oregonstate/cs361/battleship/Hard.java [hunk 1]
```diff
@@ -87,6 +87,8 @@ else if(direction.equals("vertical")){ //will be placing vertically, only rows a
     /*bojack: this function may end up being in a different class.*/
     /*wongnich: I was thinking about that, but the easy mode and hard mode have different ship placement attributes*/
     //places ships randomly
+
+    //I think this may need to be the same name as the function in battleshipmodel, what do you guys think?
     public void place() {
         Random random = new Random();
         String x, y;
```
Comment (daviandr) [inline src/main/java/edu/oregonstate/cs361/battleship/Hard.java:94]: Hey, the line below this one, 95, uses a magic number 10. Even though you have a constant at the top of the code here. You should use that constant.

**Comment by jakevolkman** [comment]
You just removed a bracket... And it breaks the function. What is this?

File: src/main/java/edu/oregonstate/cs361/battleship/Hard.java (hunk 1 above)
Comment (nickywongdong) [reply src/main/java/edu/oregonstate/cs361/battleship/Hard.java:94]: Oh thanks mang

Comment (daviandr) [reply src/main/java/edu/oregonstate/cs361/battleship/Hard.java:94]: Get your stuff together.

**Pull request closed without merging**